lib/driver.js

```diff
@@ -389,6 +390,14 @@ class XCUITestDriver extends BaseDriver {
         }
       }
 
+      try {
+        const idb = new IDB({udid});
```

Comment:
Sorry I missed this over the weekend.

Why is this happening here? This twiddling with the internals of the device object is not the correct way about things. Why is there no `initializeIdb` function on `appium-ios-simulator`? Or an added step in the `run` function?

Reply:
I didn't want to make idb private to ios-simulator, but rather use the same approach we use for android, where the instance is initiated by the android driver instance. This makes sense to me, because then we could potentially reuse the same instance for real devices without modifying the driver code (since idb supports both simulators and real devices).